Cutelyst/Plugins/Session/session_p.h

```diff
@@ -57,14 +57,14 @@ class SessionPrivate
 
     Session *q_ptr;
 
-    qint64 sessionExpires;
-    qint64 expiryThreshold;
+    qint64 sessionExpires = 7200;
```

Comment:
All of these are useless as they are going to be overwritten on setup()

Reply:
yeah, but only if setup() is really called. To silence warnings from static code checkers, you normally strive to initialize everything inside the constructor.